src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -856,7 +856,7 @@ private ICompoundAssignmentExpression CreateBoundCompoundAssignmentOperatorOpera
             BinaryOperatorKind operatorKind = Helper.DeriveBinaryOperatorKind(boundCompoundAssignmentOperator.Operator.Kind);
             Lazy<IOperation> target = new Lazy<IOperation>(() => Create(boundCompoundAssignmentOperator.Left));
             Lazy<IOperation> value = new Lazy<IOperation>(() => Create(boundCompoundAssignmentOperator.Right));
-            bool isLifted = boundCompoundAssignmentOperator.Type.IsNullableType();
+            bool isLifted = boundCompoundAssignmentOperator.Operator.Kind.IsLifted();
```

Comment:
This seems to be correctly computed using `OperatorKind.IsLifted()` at other places in this file, only this instance was different.

Reply:
👍

Reply:
Should this change affect a test output?

Reply:
I don't think so - this was just an observation from @AlekseyTs during a previous PR.

Reply:
Seems like a test gap. Could you add one that checks this property?